get-started-with-tidb-lightning.md

```diff
@@ -83,16 +95,14 @@ The TiDB Lightning installation package is included in the TiDB Toolkit. To down
     pd-addr = "172.16.31.3:2379"
     ```
 
-4. After configuring the parameters properly, use a `nohup` command to start the `tidb-lightning` process. If you directly run the command in the command-line, the process might exit because of the SIGHUP signal received. Instead, it's preferable to run a bash script that contains the `nohup` command:
+2. Run `tidb-lightning`. To avoid the program exiting due to the `SIGHUP` signal when starting the program directly in the command line using `nohup`, it is recommended to put the `nohup` command in a script. An example is provided below:
```

Comment:
```suggestion
2. Run `tidb-lightning`. To avoid the program exiting due to the `SIGHUP` signal when starting the program directly in the command line using `nohup`, it is recommended to put the `nohup` command in a script. For example:
```